README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
 ---
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Fern
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+**Fern** is an **open source toolset** for **defining, building and generating APIs and SDKs**. Fern can be used in your existing projects through OpenAPI, and integrate effortlessly into your existing workflows. It can also serve as the basis of your API development, by describing your endpoints in a Fern specification, and having Fern generate your API backend code for you.
```

Comment:
let's call it an "open source toolkit" instead of toolset (more common term)

Reply:
Alternative phrasing suggestion:Fern is an open source toolkit for building API documentation and SDKs. Bring your existing OpenAPI definition or describe your endpoints in the Fern specification. Fern automatically generates API documentation, client libraries, and backend code.

Reply:
> let's call it an "open source toolkit" instead of toolset (more common term)

Agreed. I do want to emphasize Fern's ability to generate backend code as well, what do you think of this updated version?